engine/src/gfx/mesh_gfx.cpp

```diff
@@ -455,11 +455,10 @@ Mesh::~Mesh() {
     } else {
         orig->refcount--;
         VS_LOG(debug, (boost::format("orig refcount: %1%") % refcount));
-        // TODO: Comment out?
```

Comment:
Why should this code be commented ?
Is the refcounting not properly working, useless ?
Why not removing the code instead of commenting ?

Reply:
I can remove it instead of commenting it out.

The idea here is that the `original` copy of each Mesh should never be unloaded from memory until the game shuts down. If any Mesh originals are removed, then they might have to be reloaded from disk later. And then what is the point of pre-loading everything at startup?

Reply:
That would make a nice explanation for the commit message. Can you please `git commit --amend` to add it ?

Reply:
but that would require a `rebase -i`, so I'll let you decide if you want to do it...

Reply:
Deleted the commented-out code, and provided a more detailed explanation in the corresponding commit message.

Reply:
Nice way out of a rebase, kudos !
Hey, commenting it was in fact actually useful... ;-)